nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -622,29 +695,127 @@ buildSubGraphs(const TfOpNodes &nodes, const TfOpIdxMap &map,
 
 } // namespace
 
+TfOpNodes buildRealizedOpNodes(TfOpNodes &nodes,
```

Comment:
Is it only for the bn fusing? if it is, it is better to change the function name.

Reply:
Later. i will add more fusing path in this function.
it is better to change function name for now ?